packages/ethersjs-provider/src/Web3AsyncSendable.ts

```diff
@@ -0,0 +1,38 @@
+import { queue, retryable } from "async";
+import { AsyncSendable } from "ethers/providers/web3-provider";
+import Web3 from "web3";
+
+export class Web3AsyncSendable implements AsyncSendable {
+    private asyncQueue: any;
+
+    constructor(
+        private ethNodeUrl: string,
+        times: number, 
+        interval: number, 
+        concurrency: number
+    ) 
+    {
+        const send: any = function(task: any, callback: any) {
```

Comment:
Should this callback have the same type as the one in the public send's parameters?

Reply:
Yep! Will update.